pkg/ccl/backupccl/restore_span_covering.go

```diff
@@ -355,6 +355,18 @@ func generateAndSendImportSpans(
 			Span: lastCovSpan,
 		}
 
+		// Do not send covering entry if the entire span is below the
+		// lowWaterMark.
```

Comment:
this patch logically makes sense to me. @dt @adityamaru do you see anything wrong with skipping processed `restoreSpanEntries` instead of unprocessed `requiredSpans`?